### PR TITLE
Create IslamNews.ru.xml

### PR DESCRIPTION
**Pull request by 6u621jpq6ybdw**

### CODE REVIEW

**Comment by J0WI** [comment]
Please have a look at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

File: src/chrome/content/rules/IslamNews.ru.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+	Invalid Security Certificate:
+		hadith.islamnews.ru
```
Comment (J0WI) [inline src/chrome/content/rules/IslamNews.ru.xml]: + `koran.islamnews.ru`

**Comment by 6u621jpq6ybdw** [comment]
@J0WI Added.

**Pull request merged**